presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -123,6 +124,12 @@ public SystemSessionProperties(
                         "Compute hash codes for distribution, joins, and aggregations early in query plan",
                         featuresConfig.isOptimizeHashGeneration(),
                         false),
+                booleanSessionProperty(
+                        DISTRIBUTED_JOIN,
+                        "Use a distributed join instead of a broadcast join",
+                        featuresConfig.isDistributedJoinsEnabled(),
```

Comment:
There mustn't be a default value, otherwise it's always set, always taking precedence.

Reply:
There has to be unless we get rid of the config property.  Instead I had it just take precedence if it's set to broadcast, since that's different from the system default.

Reply:
I am for dropping this as the config property, retaining only session level switch. @martint @mbasmanova ?

Reply:
@findepi I think session-level property is sufficient.